frontend/vite.config.mts

```diff
@@ -3,9 +3,11 @@ import { fileURLToPath, URL } from 'node:url';
 import { defineConfig } from 'vite';
 import vue from '@vitejs/plugin-vue';
 import { visualizer } from 'rollup-plugin-visualizer';
+//import { viteSingleFile } from "vite-plugin-singlefile"
```

Comment:
Remove?

Reply:
I was looking at reducing loading times on slow connections

The vite-plugin-singlefile is nice, but you can't control the ordering.

Ideally I'd have the CSS, then the page HTML, then the ~500kb blob of Javascript (200kb compressed) all in one file, that way the client loads the CSS and we can have a nice styled 'loading page' in the default HTML, then when the rest of the app loads the app appears. I've found this works really well in practice as long as you can keep the JS file size down to ~200-300kb (compressed).

I'll see if I can get it down to just 2 files, so CSS is inlined in the HTML file, then the JS is loaded separately.